src/components/tables/hovered_table_store.ts

```diff
@@ -48,9 +44,25 @@ export class HoveredTableStore extends SpreadsheetStore {
     if (!table) {
       return;
     }
-    const { left, right } = table.range.zone;
+    const { left, right, top } = table.range.zone;
+    const isTableHeader = row < top + table.config.numberOfHeaders;
+    const isTableRowHasContent = (): boolean => {
+      for (let c = left; c <= right; c++) {
```

Comment:
nitpick: could be shortened with `range(left, right).some((col) => !isHidden && cell.evaluatedValue)`